exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/BaseRepeatedValueVector.java

```diff
@@ -123,6 +123,15 @@ public int getBufferSize() {
   }
 
   @Override
+  public int getBufferSizeFor(int valueCount) {
+    if (getAccessor().getValueCount() == 0) {
+      return 0;
+    }
+
+    return offsets.getBufferSizeFor(valueCount) + vector.getBufferSizeFor(valueCount);
```

Comment:
This first one should be passed valueCount +1 right? As was done in VariableLengthVectors.java